cgo_flags.go

```diff
@@ -7,14 +7,10 @@ package rocksdb
 // #cgo CPPFLAGS: -Iinternal/utilities/merge_operators/string_append
 // #cgo windows CPPFLAGS: -DOS_WIN
 // #cgo !windows CPPFLAGS: -DROCKSDB_PLATFORM_POSIX -DROCKSDB_LIB_IO_POSIX
-// #cgo darwin CPPFLAGS: -DOS_MACOSX
-// #cgo linux CPPFLAGS: -DOS_LINUX -fno-builtin-memcmp -DROCKSDB_FALLOCATE_PRESENT -DROCKSDB_MALLOC_USABLE_SIZE
+// #cgo darwin CPPFLAGS: -DOS_MACOSX -DROCKSDB_BACKTRACE
+// #cgo linux CPPFLAGS: -DOS_LINUX -fno-builtin-memcmp -DROCKSDB_FALLOCATE_PRESENT -DROCKSDB_MALLOC_USABLE_SIZE -msse -msse4.2
```

Comment:
Can you verify the sse code is getting compiled? The easiest way is to edit `internal/util/crc32c.c` and add a #error to the `#ifndef __SSE4_2__` code path.